generator/Makefile

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 The Prometheus Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MIBDIR := ~/.snmp/mibs
+
+APC_URL        := 'https://download.schneider-electric.com/files?p_File_Name=powernet426.mib'
+CISCO_URL      := 'ftp://ftp.cisco.com/pub/mibs/v2/v2.tar.gz'
+SERVERTECH_URL := 'ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3.mib'
+PALOALTO_URL   := 'https://www.paloaltonetworks.com/content/dam/pan/en_US/assets/zip/technical-documentation/snmp-mib-modules/PAN-MIB-MODULES-7.0.zip'
+
+
+.DEFAULT: all
+
+.PHONY: all
+all: mibs
+
+clean:
+	@rm -rv $(MIBDIR)
```

Comment:
I don't think it's wise to blow away the user's mibdir like this, it could contain other mibs.

Reply:
Sounds reasonable.